ledger/participant-integration-api/src/main/scala/platform/usermanagement/PersistentUserManagementStore.scala

```diff
@@ -226,4 +230,8 @@ class PersistentUserManagementStore(
     rights.take(5).mkString("", ", ", closingBracket)
   }
 
+  private def epochMicroseconds(): Long = {
+    val now = Instant.now()
```

Comment:
Ledger API server classes generally use a `TimeProvider` to read the "current time" (which can serve a static time).

Could you add a comment explaining why this one uses an independent time source, and/or parametrize the store with a time source and set the time source somewhere higher up (e.g., in `ApiServices`)?

Note: There are other places that _don't_ use `TimeProvider`, like the [submission time of commands](https://github.com/digital-asset/daml/blob/main/ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala#L88). It's useful to document these exceptions.

Reply:
Using `TimeProvider` now. Can you have another look?